src/app/components/library/library.ui

```diff
@@ -9,14 +9,26 @@
         <property name="vscrollbar-policy">always</property>
         <property name="min-content-width">250</property>
         <property name="child">
-          <object class="GtkFlowBox" id="flowbox">
-            <property name="margin-start">8</property>
-            <property name="margin-end">8</property>
-            <property name="margin-top">8</property>
-            <property name="margin-bottom">8</property>
-            <property name="min-children-per-line">1</property>
-            <property name="selection-mode">none</property>
-            <property name="activate-on-single-click">0</property>
+          <object class="GtkOverlay" id="overlay">
+            <child>
+              <object class="GtkFlowBox" id="flowbox">
+                <property name="margin-start">8</property>
+                <property name="margin-end">8</property>
+                <property name="margin-top">8</property>
+                <property name="margin-bottom">8</property>
+                <property name="min-children-per-line">1</property>
+                <property name="selection-mode">none</property>
+                <property name="activate-on-single-click">0</property>
+              </object>
+            </child>
+            <child type="overlay">
+              <object class="AdwStatusPage" id="status_page">
+                <property name="title" translatable="yes" comments="A title that is shown when the user has not saved any albums.">You have no saved albums</property>
```

Comment:
perfect with comments and everything :) I'll add the strings on POEditor for translators. do you mind just ending with a period?

Reply:
Ok